### ISSUE TITLE
new short paper on software citation

### ISSUE THREAD

**Issue by danielskatz** [force11/force11-sciwg#109]
http://arxiv.org/abs/2103.06681

**Comment by sdruskat** [comment]
Should this go in the outputs list in the README? What is our policy regarding the inclusion of works published by a subgroup of this group?

**Comment by danielskatz** [comment]
I don't think this is really a group output, but if we had a list of other interesting/topical papers, we could put it there.
